appserver/admingui/plugin-service/src/main/java/org/glassfish/admingui/plugin/ConsolePluginService.java

```diff
@@ -323,8 +322,7 @@ public Map<String, List<URL>> getResources(String name) {
                     } catch (Exception ex) {
                         // Ignore b/c this should not ever happen, we're not
                         // changing the URL
-                        System.out.println(
```

Comment:
I believe this is an ERROR message, not INFO. And it would be good to log the stacktrace, especially if comment says that it should not happen :-)

```
logger.log(Level.SEVERE, "ConsolePluginService: URL Copy Failed!, ex");
```